new-lamassu-admin/src/components/expandable-table/ExpTable.js

```diff
@@ -41,7 +41,8 @@ const ExpRow = ({ id, columns, details, sizes, expanded, className, expandRow, .
           <button onClick={() => expandRow(id)} className={classes.expandButton}>
             {expanded && <ExpandOpenIcon />}
             {!expanded && <ExpandClosedIcon />}
-          </button></Td>
+          </button>
+        </Td>
       </Tr>
       <Tr className={classnames(detailsRowClasses)}>
         <Td size={mainWidth}>
```

Comment:
rowStates logic is awfully complex here, if we need to keep only one row open why not just have a id state with the open component? Then we don't have to keep creating a new list and mapping everything on that list to false.